electron/index.ts

```diff
@@ -72,6 +76,16 @@ function createWindow(connectionAtShutdown?: SerializableConnection) {
     mainWindow = null;
   });
 
+  const minimizeWindowToTray = (event: Event) => {
```

Comment:
i guess it's all the same but any reason to go with an arrow function here vs. a regular function?

Reply:
Seemed more fitting to me than a regular function since it is only used here and it is small enough. Happy to define it if you think it makes the code more readable.

Reply:
No strong feelings!